### ISSUE TITLE
Selector not working with config.container in Instantiation

### ISSUE THREAD

**Issue by jlmakes** [jlmakes/scrollreveal.js#289]
Seems like ScrollReveal is failing to recognize selectors when overriding defaults during instantiation.

_Selector as `config.container` for instantiation (broken):_
```js
var sr = new ScrollReveal({ container: '.main' });
sr.reveal('.child');
```
[JSBin Demo](https://jsfiddle.net/68hrp3hk/)

***

_DOM Node as `config.container` for instantiation (works):_
```js
var sr = new ScrollReveal({ container: document.querySelector('.main') });
sr.reveal('.child');
```
[JSBin Demo](https://jsfiddle.net/L4gvhrnw/)

***

_Selector as `config.container` for reveal (works):_
```js
var sr = new ScrollReveal();
sr.reveal('.child', { container: '.main' });
```
[JSBin Demo](https://jsfiddle.net/dmp5pz3s/)

**Comment by amanjareck** [comment]
Yes, thats why I was banging my head against the wall (lol.. jk).  Thanks for the solution though works like a charm :)

**Comment by jumbosushi** [comment]
Hello! I decided to take a stab at this in the spirit of hacktoberfest

**Comment by jumbosushi** [comment]
Hello! I decided to take a stab at this in the spirit of hacktoberfest.
The problem seemed to be how _resolveContainer never saved the string selector as Node object in sr.defaults.container, so I made a PR to fix it at #302!
Let me know if you want me to change anything

**Comment by jumbosushi** [comment]
https://github.com/jlmakes/scrollreveal.js/pull/302
